src/app/assets/themes/kwanzaa/flag-cursor.svg

```diff
@@ -0,0 +1,15 @@
+<svg width="50" height="28" viewBox="0 0 50 28" fill="none" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+<g clip-path="url(#clip0)">
+<rect width="50" height="28" fill="white"/>
+<rect width="50" height="28.125" fill="url(#pattern0)"/>
+</g>
+<defs>
+<pattern id="pattern0" patternContentUnits="objectBoundingBox" width="1" height="1">
+<use xlink:href="#image0" transform="scale(0.000520833 0.000925926)"/>
```

Comment:
Is this an image but pasted into an SVG file directly?

Reply:
Yeah, I resized it in figma and exported as an svg. Didn't think anything of it, is this a concern? I'm not totally familiar with all the filetype differences.

Reply:
It usually is a file size concern. SVG gives you vector graphs without pixelations; png is bitmap.

Reply:
PNG was significantly smaller in size so I swapped the cursor image out.